### DISCUSSION TITLE
How to contibute gssoc extended within 4 days

### DISCUSSION THREAD

**Discussion by aimanrazia** [recodehive/Stackoverflow-Analysis#451] [Q&A] [answered]
I am new and just i got about to know about the gssoc can anyone help me tips provide step by step

**Comment by sanjay-kv** [comment]
https://stream.recodehive.com/github

check this out

**Comment by SanyukthaPrabhu** [comment, marked as answer]
Bro better to go through once with these playlists https://academy.girlscript.tech/gssoc 
its very basic and explained well...Thanks to @sanjay-kv
